be/src/agent/agent_task.cpp

```diff
@@ -500,7 +500,7 @@ void run_compaction_task(const std::shared_ptr<CompactionTaskRequest>& agent_tas
     for (auto tablet_id : compaction_req.tablet_ids) {
         EngineManualCompactionTask engine_task(GlobalEnv::GetInstance()->compaction_mem_tracker(), tablet_id,
                                                compaction_req.is_base_compaction);
-        StorageEngine::instance()->execute_task(&engine_task);
+        (void)StorageEngine::instance()->execute_task(&engine_task);
```

Comment:
What's the usage of `(void)`

Reply:
CI will enable [[nodiscard]] check for Status return value soon. We must handle all status return in the code explicitly. (void) Means that it is safe to ignore the status and prevent the warning from the complier.